glotaran/utils/io.py

```diff
@@ -164,6 +164,9 @@ def load_datasets(dataset_mappable: DatasetMappable) -> DatasetMapping:
 def relative_posix_path(source_path: StrOrPath, base_path: StrOrPath | None = None) -> str:
     """Ensure that ``source_path`` is a posix path, relative to ``base_path`` if defined.
 
+    On windows if ``source_path`` and ``base_path`` are on different drives, it will return
```

Comment:
windows -> Windows

Reply:
Fixed in 4d6301c0a66e31de369eba0c044c162f2f81c60e